.github/PULL_REQUEST_TEMPLATE/add-preview-link.md

```diff
@@ -0,0 +1,14 @@
+<!--
+If this PR modifies the website (docs/content/** or a file included from there), then please include in your opening comment a line of the following form:
+
+Preview: https://<your-gh-username>.github.io/<your-fork-name>/<your-branch-name>
+
+If your branch name starts with doc-, the preview will update automatically on each push.
```

Comment:
The accurate condition for this IF also includes the contributor's fork having a properly set up `gh-pages` branch.